internal/app/network/settings.go

```diff
@@ -65,13 +34,15 @@ func FetchSettings(client *http.Client) (*DefaultSettings, error) {
 }
 
 type DefaultSettings struct {
-	ProjectName    StringParam `json:"project_name"`
-	CompanyWebsite StringParam `json:"company_website"`
-	Engine         EnumParam   `json:"engine"`
-	KtorVersion    EnumParam   `json:"ktor_version"`
-	KotlinVersion  EnumParam   `json:"kotlin_version"`
-	BuildSystem    EnumParam   `json:"build_system"`
-	ConfigType     EnumParam   `json:"configuration_in"`
+	ProjectName      StringParam `json:"project_name"`
+	CompanyWebsite   StringParam `json:"company_website"`
+	Engine           EnumParam   `json:"engine"`
+	KtorVersion      EnumParam   `json:"ktor_version"`
+	KotlinVersion    EnumParam   `json:"kotlin_version"`
+	BuildSystem      EnumParam   `json:"build_system"`
+	ConfigType       EnumParam   `json:"configuration_in"`
+	CompileTargets   EnumParam   `json:"compile_targets"`
+	ProjectStructure EnumParam   `json:"project_structure"`
```

Comment:
I have changed my mind, so now the project structure and compile targets will be determined solely from the selected plugins.  These will need to be removed 😄